pre_commit_hooks/go/golangci-lint.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 if hash golangci-lint 2>/dev/null; then
-    golangci-lint run --enable-all --fix "$@"
+    golangci-lint run -v --fix "$@"
```

Comment:
This enables verbose output per default which helps to understand the existing issues and how they are being fixed.
<img width="2951" alt="Screenshot 2020-07-13 at 11 25 43" src="https://user-images.githubusercontent.com/930643/87288422-9c84b680-c4fb-11ea-83d7-f87f4003841c.png">